### ISSUE TITLE
pre-trained weights of the IHC classifier

### ISSUE THREAD

**Issue by GDC-YYDS** [jiahanli2022/confusion-GAN#5]
Your work on confusion-GAN and the integration of weakly-supervised pathology information is very inspiring for my current research.
    I noticed that the pre-trained IHC classifier, which plays a key role in the pathology loss，I would like to kindly ask if you could share the pre-trained weights of the IHC classifier and A _lables to help me better reproduce your results.

**Comment by yeori102** [comment]
Hi, I’m also trying to reproduce the results of Confusion-GAN and ran into a similar issue.

It seems that pretrain_IHC_classifier.pth is required to run train.py, but it is not included in the repository. I noticed your previous question about this, so I was wondering how you handled the pretrained IHC classifier.

Were you able to obtain the pretrained IHC_classifier.pth from the authors, or did you train the classifier yourself?

Thanks in advance!

**Comment by GDC-YYDS** [comment]
我是自己预训练了一个分类器，你可以按照论文中的描述用AI写一个cnn网络用来判定阳性和阴性，并在数据集上预训练，得到权重后按照论文中的描述，新增加一个判别器，训练生成器，
